_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -0,0 +1,104 @@
+---
+id: akn7bn
+name: iframe with negative tabindex has no interactive elements
+rule_type: atomic
+description: |
+  This rule checks that `iframe` elements with a negative `tabindex` attribute value contain no interactive elements.
+accessibility_requirements:
+  wcag20:2.1.1: # Keyboard (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  wcag-technique:G202: # Ensuring keyboard control for all functionality
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Brian Bors
+    - Wilco Fiers
+---
+
+## Applicability
+
+This rule applies to any `iframe` element that has a negative number as a `tabindex` [attribute value][].
+
+## Expectation
+
+For each test target, the [nested browsing context][] does not include elements that are [visible][] and part of the [sequential focus navigation][]. An element is "included" in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][]. 
```

Comment:
Wouldn't that first line be more appropriate in the applicability section? At the moment the applicability says all `<iframe tabindex="-1">` whereas it's more about `<iframe tabindex="-1">` with widgets.

Reply:
I'm not sure. I suppose it can be done in a bunch of different ways. We could also completely flip it around and make this applicable to any focusable element in an iframe, and then have an expectation that the iframe doesn't have a tabindex.... I think I prefer it this way around, but if you have a strong preference I don't mind changing it.

Reply:
No strong preference. Valid point.